packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3836,6 +3837,24 @@ export type ResourceDetails = {
     resourceUrl: string;
 };
 
+// @beta
+export interface RichTextEditorOptions {
+}
```

Comment:
We do we foresee that could go into this options? Aka from the composite level, do we normally leave that much flexibilities to contosos? Or we simply only let contoso to decide whether they want rich text or not?

Reply:
Same as above, this meant more of stub and would likely be changed.